cmd/edenClean.go

```diff
@@ -40,7 +40,7 @@ var cleanCmd = &cobra.Command{
 			redisPort = viper.GetInt("redis.port")
 			redisDist = utils.ResolveAbsPath(viper.GetString("redis.dist"))
 			eveImageFile = utils.ResolveAbsPath(viper.GetString("eve.image-file"))
-			evePidFile = utils.ResolveAbsPath(viper.GetString("eve.pid"))
+			evePidFile = utils.ResolveAbsPath(configName + "-" + viper.GetString("eve.pid"))
```

Comment:
Probably, we need the same [here](https://github.com/lf-edge/eden/blob/22f78cc2c39181edc9ca69973fa67c2fc4c3311a/cmd/eve.go#L58) and [here](https://github.com/lf-edge/eden/blob/22f78cc2c39181edc9ca69973fa67c2fc4c3311a/cmd/eve.go#L147)?

Reply:
The [same](https://github.com/lf-edge/eden/blob/22f78cc2c39181edc9ca69973fa67c2fc4c3311a/cmd/eve.go#L217)

Reply:
Probably - it needs to be seen.

Reply:
Fixed

Reply:
Seems -- some Viper's black magic:
```
$ ./eden start -v debug
...
FATA[0000] open /home/user/work/EVE/github/itmo-eve/eden/dist/default-/home/user/work/EVE/github/itmo-eve/eden/dist/default-eve.log: no such file or directory 
ERRO[0014] cannot start eve: exit status 1
```
Need to look...